services/web/server/src/simcore_service_webserver/login/settings.py

```diff
@@ -51,6 +51,11 @@ class LoginSettings(BaseCustomSettings):
         description="If true, it enables two-factor authentication (2FA)",
     )
 
+    LOGIN_PASSWORD_MIN_LENGTH: PositiveInt = Field(
```

Comment:
Do not forget to add  (⚠️ devops) in the title to mark there is a new environ

Reply:
oha good catch thx. For now we will leave it on the default of 12, but yes, env-var changes should in principle have a warning indeed :)